test/dynamics/models/test_rotating_frame.py

```diff
@@ -534,75 +539,76 @@ def test_state_transformations_no_frame_Operator_types(self):
         rotating_frame = RotatingFrame(None)
 
         t = 0.123
-        y = Operator([1.0, 1j])
+        y = self.asarray(Operator([1.0, 1j]))
         out = rotating_frame.state_into_frame(t, y)
         self.assertAllClose(out, y)
-        self.assertTrue(isinstance(out, np.ndarray))
+        self.assertEqual(DYNAMICS_NUMPY_ALIAS.infer_libs(out)[0], self.array_library())
         out = rotating_frame.state_out_of_frame(t, y)
         self.assertAllClose(out, y)
-        self.assertTrue(isinstance(out, np.ndarray))
+        self.assertEqual(DYNAMICS_NUMPY_ALIAS.infer_libs(out)[0], self.array_library())
 
         t = 100.12498
-        y = Operator(np.eye(2))
+        y = self.asarray(Operator(np.eye(2)))
         out = rotating_frame.state_into_frame(t, y)
         self.assertAllClose(out, y)
-        self.assertTrue(isinstance(out, np.ndarray))
+        self.assertEqual(DYNAMICS_NUMPY_ALIAS.infer_libs(out)[0], self.array_library())
         out = rotating_frame.state_out_of_frame(t, y)
         self.assertAllClose(out, y)
-        self.assertTrue(isinstance(out, np.ndarray))
+        self.assertEqual(DYNAMICS_NUMPY_ALIAS.infer_libs(out)[0], self.array_library())
 
     def test_state_transformations_no_frame_array_type(self):
         """Test frame transformations with no frame."""
 
         rotating_frame = RotatingFrame(None)
 
         t = 0.123
-        y = Array([1.0, 1j])
+        y = self.asarray([1.0, 1j])
```

Comment:
I think just using numpy is fine
```suggestion
        y = np.asarray([1.0, 1j])
```